htdocs/compta/compta-files.php

```diff
@@ -75,24 +86,24 @@
 if ($user->societe_id > 0)
     accessforbidden();
 
-
-
 /*
  * Actions
  */
-
 //$parameters = array('socid' => $id);
 //$reshook = $hookmanager->executeHooks('doActions', $parameters, $object); // Note that $object may have been modified by some hooks
 //if ($reshook < 0) setEventMessages($hookmanager->error, $hookmanager->errors, 'errors');
 
+/*
+ * Fetch the lines/files from db /
+ */
 $filesarray=array();
 $result=false;
 if(($action=="searchfiles"||$action=="dl" ) && $date_start && $date_stop){
     $wheretail=" '".$db->idate($date_start)."' AND '".$db->idate($date_stop)."'";
-    $sql="SELECT rowid as id, ref as ref,paye as paid, total_ttc, fk_soc, datef as date, 'Invoice' as item FROM ".MAIN_DB_PREFIX."facture";
+    $sql="SELECT rowid as id, facnumber as ref,paye as paid, total_ttc, fk_soc, datef as date, 'Invoice' as item FROM ".MAIN_DB_PREFIX."facture";
```

Comment:
This field does not exists anymore in develop version

Reply:
I'll check how I can remove the ref from the SQL because I name display is managed by the getNom function (most likely only  the log part need to be adapted)